bio/dada2/dereplicate-fastq/meta.yaml

```
@@ -0,0 +1,11 @@
+name: dada2_dereplicate_fastq
+description: |
+  `DADA2 <https://benjjneb.github.io/dada2/index.html>`_
+  Dereplication of FASTQ files using dada2 derepFastq function. Optional parameters are documented in the `manual <https://www.bioconductor.org/packages/release/bioc/manuals/dada2/man/dada2.pdf>`_.
+authors:
+  - Charlie Pauvert
+input: a FASTQ file
+params:
+  - R optional arguments passed to the wrapped function as named Python arguments
```

I'm not sure how clear this info is, maybe rephrase to something like:
```suggestion
  - optional arguments for derepFastq(), please provide them as python key=value pairs
```

---

This is nicer thanks! It's included.
But on a side note, these information in the `meta.yaml` do not show up in the documentations AFAIK.

---

Hmm, we should double-check this. You can compile the docs locally and have a look. There's a quick walk-through in the lower part of this contribution docs section:
https://snakemake-wrappers.readthedocs.io/en/stable/index.html#testing-locally

I would have thought, that it simply renders all `meta.yaml` entries as subheadings, but it might be that only certain names are rendered...

---

Yes thanks I had already did built locally. I have a WIP branch and some suggestions in mind. I might list it properly as an issue though.